buildSrc/src/gen-execution.kt

```diff
@@ -5,6 +5,151 @@ import org.intellij.lang.annotations.Language
 import java.io.File
 
 fun LangData.execution(nickname: String, configName: String, outDir: File) {
+	@Language("java")
```

Comment:
Use uppercase "JAVA". This annotation is for highlighting the following string block as Java code

Reply:
interesting, mine still highlights with lowercase java

Reply:
Ok, than I think it's fine to leave it.